crates/sui-protocol-config/src/lib.rs

```diff
@@ -1414,6 +1433,11 @@ impl ProtocolConfig {
                 24 => {
                     cfg.feature_flags.simple_conservation_checks = true;
                     cfg.max_publish_or_upgrade_per_ptb = Some(5);
+
+                    // TODO(tzakian)[tto] This should only be set in the protocol version that we
+                    // release with.
+                    cfg.transfer_receive_object_cost_base = Some(52);
+                    cfg.feature_flags.receive_objects = true;
```

Comment:
Do we want to turn it on like this yet? I had thought we only wanted it on for devnet initially?

Reply:
This is just what I have for testing ATM, but you're totally correct. I'll update to so this is only turned on for devnet.